k8s/sandbox/common/idam-sprod/idam-api.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: idam-api
+  namespace: idam-sprod
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:preview-*
+spec:
+  releaseName: idam-api
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: idam-api
+    version: 0.2.4
+  values:
+    java:
+      image: hmctspublic.azurecr.io/idam/api:preview
+      ingressHost: idam-api-sbox.service.core-compute-sprod.internal
+      replicas: 3
+      aadIdentityName: idam-sprod
+      environment:
```

Comment:
This requires more environment overrides: 
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/idam/idam-api.yaml#L28-L30

Reply:
done